pkg/util/util_test.go

```diff
@@ -0,0 +1,50 @@
+package util
+
+import (
+	"testing"
+)
+
+func Test_isValidHostname(t *testing.T) {
+	tests := []struct {
+		hostname string
+		want     bool
+	}{
+		{
+			// Starts with an number.
+			hostname: "1solana.com",
+			want:     true,
+		},
+		{
+			// Ends with an number.
+			hostname: "solana.com1",
+			want:     true,
+		},
+		{
+			// Starts with an underscore.
+			hostname: "_solana.com",
+			want:     false,
+		},
+		{
+			// Ends with an underscore.
+			hostname: "solana.com_",
+			want:     false,
+		},
+		{
+			// No TLD.
+			hostname: "solana",
+			want:     false,
+		},
+		{
+			// With TLD.
+			hostname: "solana.com",
+			want:     true,
+		},
+	}
+	for _, tt := range tests {
```

Comment:
Just a stylistic preference, but what do you think of just using an anonymous struct here?

```suggestion
	for _, tt := range []struct {
		hostname string
		want     bool
	}{
		{
			// Starts with an number.
			hostname: "1solana.com",
			want:     true,
		},
		{
			// Ends with an number.
			hostname: "solana.com1",
			want:     true,
		},
		{
			// Starts with an underscore.
			hostname: "_solana.com",
			want:     false,
		},
		{
			// Ends with an underscore.
			hostname: "solana.com_",
			want:     false,
		},
		{
			// No TLD.
			hostname: "solana",
			want:     false,
		},
		{
			// With TLD.
			hostname: "solana.com",
			want:     true,
		},
	} {
	... snip ...
```

Reply:
I think we all have different styles and if the goal is to normalize then when have to be less relaxed and put linting to the work. For now, this is how VSCode generates unit test functions so I'd like to keep the current format as it will help speed up adding tests to this codebase.

Reply:
TIL: vscode can automatically create Table Driven test cases. I just type them out manually, like we were meant to!